src/node-lib/curriculum-api-2023/queries/unitListing/unitListing.query.ts

```diff
@@ -19,9 +19,13 @@ const unitListingQuery =
       return null;
     }
 
-    const parsedRawUnits = unitsForProgramme.map((p) =>
-      syntheticUnitvariantLessonsSchema.parse(p),
-    );
+    // For some reason the parse is parsing out subject parent
```

Comment:
what does this mean?

Reply:
I can't access the student_parent after the `syntheticUnitvariantLessonsSchema.parse(p)` it gets filtered out the programme_fields obj

Reply:
fixed ✅